numba/tests/test_parfors.py

```diff
@@ -3363,6 +3363,18 @@ def f(r):
         r = np.array([[0., 0., 0.], [0., 0., 1.]])
         self.assertPreciseEqual(f(r), f.py_func(r))
 
+    @skip_parfors_unsupported
+    def test_issue6774(self):
+        @njit(parallel=True)
+        def test_impl():
+            na_mask = np.ones((5,))
+            result = np.empty((5,))
+            for i in prange(1):
+                result[i] = np.sum(na_mask[i:i + 1])
+            return result
+
+        self.check(test_impl)
```

Comment:
This test is relying on uninitialised memory in the result. Think it should be something like:
```python
@njit(parallel=True)
def test_impl():
    n = 5
    na_mask = np.ones((n,))
    result = np.empty((n - 1,))
    for i in prange(len(result)):
        result[i] = np.sum(na_mask[i:i + 1])
    return result
```
At present the single `prange` iteration will only write to the first entry of `result` and there will be random junk in the rest of it.

Reply:
Done.